src/app/pages/request/request.component.ts

```diff
@@ -41,7 +43,7 @@ export class RequestComponent implements OnInit {
   ngOnInit() {
     this.requestForm = this.formBuilder.group({
       priority: ['', Validators.required],
-      title: ['', [Validators.required, Validators.minLength(20)]],
+      title: ['', [Validators.required, Validators.maxLength(40)]],
```

Comment:
By doing this, you are forcing the title to have the max character length of 40 othewise it will fail. this is not what we want for this story though.